packages/terra-avatar/tests/nightwatch/FacilityAvatar.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+import Avatar from '../../lib/Avatar';
+
+export default () => <Avatar id="facility-avatar" variant="facility" ariaLabel="facility" />;
```

Comment:
this is cheating, but I dig it ha

Reply:
*effective coding* is a good synonym here!